babble/src/main/java/io/mosaicnetworks/babble/service/BabbleService.java

```diff
@@ -163,21 +166,24 @@ public void startArchive(final String configDirectory, GroupDescriptor groupDesc
         new Thread(new Runnable() {
             public void run() {
                 try {
-                    mBabbleNode = BabbleNode.create(new BlockConsumer() {
-                        @Override
-                        public Block onReceiveBlock(Block block) {
-                            Log.i("ProcessBlock", "Process block");
-                            Block processedBlock = mAppState.processBlock(block);
-                            notifyObservers();
-                            return processedBlock;
-                        }
-                    }, configDirectory, new NodeStateChangeHandler() {
+                    mBabbleNode = BabbleNode.create(
+                            new BlockConsumer() {
+                                    @Override
+                                    public Block onReceiveBlock(Block block) {
+                                        Log.i("ProcessBlock", "Process block");
+                                        Block processedBlock = mAppState.processBlock(block);
+                                        notifyObservers();
+                                        return processedBlock;
+                                    }
+                            },
+                            configDirectory,
+                            new NodeStateChangeHandler() {
                         @Override
                         public void onStateChanged(BabbleNode.State state) {
                             mNodeState = state;
                         }
-                    });
-
+                    }
+                    );
                 } catch (IllegalArgumentException ex) {
                     //TODO: need more refined Babble exceptions
                     if (listener!=null) {
```

Comment:
Are there any functional changes to this class? Or is this smoke and mirrors!!!?

Reply:
smoke and mirrors... just formatting code to look nicely in my editor.